src/cryptonote_core/blockchain.cpp

```diff
@@ -668,6 +674,8 @@ block Blockchain::pop_block_from_blockchain()
       }
     }
   }
+  if (pruned)
+    MWARNING(pruned << " pruned txes could not be added back to the txpool");
```

Comment:
I assume this is intended for cases of deep reorg (> 5500 blocks). Do the unpruned txes returned to the pool get re-broadcast to other nodes who may have pruned those txes?